### ISSUE TITLE
Abby2Alto

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/ocr-transform#15]
https://github.com/ironymark/AbbyyToAlto, Transformation with php, GPL v3

**Comment by kba** [comment]
Yes, I've seen it but I very much prefer a declarative transformation in XSLT that has no possible side effects and is easier to test. Maybe we can convert it to XSLT?

**Comment by zuphilip** [comment]
Yes, it would be preferable to use a XSLT for the transformation.

**Comment by zuphilip** [comment]
There is also a newer implementation with Java (+Maven): https://github.com/Mewel/abbyy-to-alto

**Comment by kba** [comment]
How does that compare with https://github.com/PRImA-Research-Lab/prima-page-converter @maxnth

**Comment by stweil** [comment]
> they include the copyrighted schema for Abbyy 10

Yes, sorry, that was the one which I meant.

**Comment by mikegerber** [comment]
> I had problems with prima-page-converter (going to open a bug report), 

https://github.com/PRImA-Research-Lab/prima-page-viewer/issues/24 - I opened the issue against prima-page-viewer as it is affected, too.

**Comment by mikegerber** [comment]
> while [Mewel/abbyy-to-alto](https://github.com/Mewel/abbyy-to-alto) worked right away.

Sort of - it does not produce `Processing` tags (or the ALTO v2 equivalent), so it is lacking too.

**Comment by stweil** [comment]
Somehow related: I just found a [converter from ABBYY to hOCR](https://github.com/stweil/archive-hocr-tools/blob/master/docs/source/index.rst#abbyy-to-hocr) made by the Internet Archive. No own tests done so far.
